docs/4.4/preview/teleport-application-access.md

```diff
@@ -0,0 +1,206 @@
+# Teleport Application Access Beta
+
+Teleport is currently beta testing an application access proxy. Teleport Application Access has been designed to secure internal web applications, letting you provide secure access while improving both visibility and control for access.
+
+Here are a few things you might want to secure with Teleport Application Access
+
+- Internal Control Panels
+- Wikis / Tooling that's only available on the VPN.
+- Access to the Kubernetes Dashboard
+- Developer tools. Such as Jenkins, or the Atlassian stack.
+
+**Example Teleport Application Usage**
+
+This diagram shows Teleport Deployed into the AWS, Teleport Application Access is now providing access Jenkins and an internal dashboard. Another Teleport Application Service is running in another Data Center and dials back to the Teleport Cluster. This enables users to access that other dashboard.
+![Example App Access Usage](/img/aap.svg)
+
+_Want to see a demo of AAP in action. [Click here](#view-applications-in-teleport)_
+## Teleport Setup
+
+Teleport Application Access requires two processes to be ran. One is the dedicated Teleport Bastion host and the other App Service will proxy the applications. The App service can be ran on the host of the app or it can be put in front of it.
```

Comment:
```suggestion
Teleport Application Access requires two processes to be run. One is the dedicated Teleport bastion host (auth/proxy service) and the other App service will proxy the applications. The App service can be run on the host of the app, or it can be put in front of it.
```